sdk/include/xdk/mmfuncs.h

```diff
@@ -646,7 +646,7 @@ NTKERNELAPI
 VOID
 NTAPI
 MmFreeMappingAddress(
-  _In_ PVOID BaseAddress,
+  _Post_invalid_ PVOID BaseAddress,
```

Comment:
Probably the most complete annotation here would be: `_In_ __drv_freesMem(Mem) _Post_invalid_`
:smile:

Reply:
OK, will apply it.

Reply:
Applied, thanks. :slightly_smiling_face: